recipes/stlab/all/conanfile.py

```diff
@@ -127,8 +127,9 @@ def validate(self):
         if self.settings.compiler == "clang" and Version(self.settings.compiler.version) < "8":
             raise ConanInvalidConfiguration("Need Clang >= 8")
 
-        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) < "15.8":
-            raise ConanInvalidConfiguration("Need Visual Studio >= 2017 15.8 (MSVC 19.15)")
+        # Actually, we want *at least* 15.8 (MSVC 19.15), but we cannot check this for now with Conan.
+        if self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version) < "15":
+            raise ConanInvalidConfiguration("Need Visual Studio >= 2017")
```

Comment:
Actually, you can by using the `msvc` compiler.
https://github.com/conan-io/conan/blob/39b1bb3d32576323927033d86fce6f641a1a8518/conans/client/migrations_settings.py#L2013-L2019

Reply:
I added a check on top of the one for Visual Studio, so that we still support Visual Studio :)

Should I change the check for VS, and make it <16 rather than 15?

Reply:
`msvc` is not used widely (yet).
So let's keep using `Visual Studio` for now.

I _think_ that there will be a migration from `Visual Studio` to msvc, but please don't quote me on this.